src/callbacks_stage/entropy_bounded_limiter.jl

```diff
@@ -0,0 +1,69 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+#! format: noindent
+
+mutable struct EntropyBoundedLimiter{RealT <: Real}
+    exp_entropy_decrease_max::RealT # < 0
+end
+
+"""
+    EntropyBoundedLimiter{RealT <: Real}(; exp_entropy_decrease_max = 1f-13)
+
+Entropy-bounded limiter by
+- Lv, Ihme (2015)
+  Entropy-bounded discontinuous Galerkin scheme for Euler equations
+  [doi: 10.1016/j.jcp.2015.04.026](https://doi.org/10.1016/j.jcp.2015.04.026)
+
+This is an ideal-gas specific limiter that bounds the (unphysical) entropy decrease per element 
+from one time step (or Runge-Kutta stage) to the next.
+The parameter `exp_entropy_decrease_max` is the maximum allowed exponentiated
+entropy decrease per element at each element's node.
+
+In the original version of the paper, this value is set to zero to
+ensure that the entropy does not decrease, i.e., guarantee entropy stability in the sense of 
+- Tadmor (1986)
+  A minimum entropy principle in the gas dynamics equations
+  [doi: 10.1016/0168-9274(86)90029-2](https://doi.org/10.1016/0168-9274(86)90029-2)
+
+This, however, leads in general to very diffusive solutions for timesteps violating 
+a CFL condition (Lemma 3 in Lv, Ihme (2015)) which is required for entropy stability in the mean values.
+Since most practical simulations will employ a significantly larger timestep, one can relax the 
+strict entropy increase requirement by setting `exp_entropy_decrease_max` to a negative value.
+The limiter acts if the exponentiated entropy decrease on an element is larger than
```

Comment:
If the decrease is larger than `exp_entropy_decrease_max`? Isn't it something like "absolutely larger than..."?

Reply:
Yeah, I struggled a lot with formulating. What I intend with this is that _decrease_ carries the $-$ sign and thus "larger decrease" is smaller numbers, larger in absolute value. I will try to spell this out somewhat less misleading.

Reply:
Yes I'm also not sure what's the best option :sweat_smile: